packages/backend-core/src/logging/pino/logger.ts

```diff
@@ -159,6 +161,16 @@ if (!env.DISABLE_PINO_LOGGER) {
     return appId
   }
 
+  const getAutomationId = () => {
+    let appId
+    try {
+      appId = context.getAutomationId()
+    } catch (e) {
+      // do nothing
```

Comment:
The reason is that this may be invoked anywhere a console statement is done. We may or may not be in automation context at this point, or in any context at all. The safest approach is to try-catch so that we don't break logging.